src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/misc/ModuleBetterTab.kt

```diff
@@ -0,0 +1,62 @@
+package net.ccbluex.liquidbounce.features.module.modules.misc
+
+import net.ccbluex.liquidbounce.config.types.Configurable
+import net.ccbluex.liquidbounce.config.types.NamedChoice
+import net.ccbluex.liquidbounce.config.types.ToggleableConfigurable
+import net.ccbluex.liquidbounce.features.module.Category
+import net.ccbluex.liquidbounce.features.module.ClientModule
+import net.ccbluex.liquidbounce.render.engine.Color4b
+import net.minecraft.client.network.PlayerListEntry
+import java.awt.Color
+
+
+/**
+ * ModuleBetterTab
+ *
+ * @author sqlerrorthing
+ * @since 12/28/2024
+ **/
+object ModuleBetterTab : ClientModule("BetterTab", Category.MISC) {
+
+    init {
+        treeAll(
+            Limits,
+            Highlight,
+            AccurateLatency
+        )
+    }
+
+    val sorting by enumChoice("Sorting", Sorting.VANILLA)
+
+    object AccurateLatency : ToggleableConfigurable(ModuleBetterTab, "AccurateLatency", true) {
+        val suffix by boolean("AppendMSSuffix", true)
+    }
+
+    object Limits : Configurable("Limits") {
+        val tabSize by int("TabSize", 80, 1..1000)
+        val height by int("ColumnHeight", 20, 1..100)
+    }
+
+    object Highlight : ToggleableConfigurable(ModuleBetterTab, "Highlight", true) {
+        class HighlightColored(name: String, color: Color4b) : ToggleableConfigurable(this, name, true) {
+            val color by color("Color", color)
+        }
+
+        val self = tree(HighlightColored("Self", Color4b(Color(50, 193, 50, 80))))
+        val friends = tree(HighlightColored("Friends", Color4b(Color(16, 89, 203, 80))))
```

Comment:
I suggest sticking to the color already used by ESP.

Reply:
i think we don't need it, this code is 4 lines of code, I don't think this is too critical copying of code.

Reply:
I don't think it's a good idea, since you still need to adjust the transparency, I would like to leave the possibility of this adjustment to the user.